environment.yml

```diff
@@ -10,7 +10,7 @@ dependencies:
   - numpy >=1.13.3
   - root >=6.18.04
   - pip:
-      - "awkward>=1.2.0"
+      - "awkward>=2.0.0"
       - "uproot==4.*"
```

Comment:
This shouldn't work? uproot has to be 5?

Reply:
If the `awkward` version has no cap, the `uproot` version should have no cap, either.

```
      - "awkward>=2.0.0"
      - "uproot>=5.0.0"
```

is consistent. (Now Uproot has a strict pip dependency on Awkward; the recommendation for people who want Uproot without Awkward is to do a `--no-deps`.)

This is also consistent:

```
      - "awkward>=1.2.0"
      - "uproot>=4.0.0"
```

if you're going for supporting both with one Vector.

Actually, Vector shouldn't have any dependency on Uproot, now that I think about it. This environment.yml must be for some notebook or other documentation, right?

Reply:
Yes, it was added for the example notebooks. I have never used it though, and I am not sure if it still works with the example notebooks.